### ISSUE TITLE
Jet Smart Filter - Listing grid - Can't use filter on archive taxinomie

### ISSUE THREAD

**Issue by Biblothecaire** [CrocoBlock/suggestions#154]
Hi,

I have create a page for smar filter, work fine : https://madatest.com/hotels/
My question is, how to do the same for the TAXINOMIE archive ?
I'm a guide, so i have a lot of destination (Paris / Antananarivo / london / etc...)
A have create a Taxinomie "destination", so i just want to create a elementor template for the taxinomie "destination", but on the "listing grid" there is no option to automatically detect the taxonomy of the archive and make the filter (as the "archive" of elementor pro, exemple : https://madatest.com/destination/antananarivo/)

To summarize, what is needed is:
1) Resume the "archive" functionality of elementor archive and add it on the list grid
2) Make" Smart Filter" Filters Take into Account the Listing Grid Rule

Thanks for your help

**Comment by Floydtm** [comment]
https://github.com/CrocoBlock/suggestions/issues/160
